shared/provision/code-page/qr-scan/container.tsx

```diff
@@ -11,7 +11,9 @@ export default () => {
   const waiting = Container.useSelector(state => WaitingConstants.anyWaiting(state, Constants.waitingKey))
   const dispatch = Container.useDispatch()
   const _onSubmitTextCode = React.useCallback(
-    () => (code: string) => dispatch(ProvisionGen.createSubmitTextCode({phrase: new HiddenString(code)})),
```

Comment:
this subtley doesn't trip of tsc, its just a function that ignores the params and returns something unused... which is annoying